package.json

```diff
@@ -22,12 +22,7 @@
     "bootstrap": "^3.3.6",
     "html5shiv": "^3.7.3",
     "jquery": "^2.2.0",
-    "jquery-placeholder": "^2.1.1",
-    "react": "^0.14.7",
-    "react-dom": "^0.14.7",
-    "respond.js": "^1.4.2",
```

Comment:
respond.js need to be included in case you have to support IE < 9.
See https://github.com/micromata/bootstrap-kickstart/blob/master/templates/default.hbs#L36